src/sheet/components/SheetDataEditorTable.tsx

```diff
@@ -63,16 +85,45 @@ export default function SheetDataEditorTable({
   }
 
   const headerClass =
-    'bg-hello-csv-muted py-3.5 pr-3 pl-4 text-left text-sm font-semibold text-gray-900 whitespace-nowrap border-y border-gray-300';
+    'bg-hello-csv-muted py-3.5 pr-3 pl-4 text-left text-sm font-semibold text-gray-900 whitespace-nowrap border-y border-gray-300 shrink-0';
   const cellClass =
-    'text-sm font-medium whitespace-nowrap text-gray-900 border-b border-gray-300 max-w-[350px]';
+    'flex text-sm font-medium whitespace-nowrap text-gray-900 border-b border-gray-300 shrink-0 relative isolate justify-start items-center';
+
+  // The scrollable element for your list
+
+  const rows = table.getRowModel().rows;
+  // The virtualizer
+  const rowVirtualizer = useVirtualizer({
+    count: rows.length,
+    getScrollElement: () => tableContainerRef.current,
+    estimateSize: () => 52.62,
```

Comment:
Where did we got this number from?

Reply:
Yeah, we should definitelly extract it to constant and add some comment. As far as I understand this, this value is not critical and if it is set incorrectly, then we would only got some problems with the scrollbar, right?